qml/pages/VenueMapOverviewPage.qml

```diff
@@ -121,10 +118,51 @@ BVApp.Page {
         }
 
         Component.onCompleted: {
+            centerAndZoom();
+        }
+
+        function centerAndZoom()
+        {
             addMapItem(currentPosition);
```

Comment:
Why do you keep adding the ´´´currentPosition´´´ item to the map
on every re-center?